<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerWindow</class>
 <widget class="QWidget" name="DebuggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebuggerWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scope</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="stream_view"/>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="delta_view"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Stats</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mean Inter-packet delta time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max Inter-packet delta time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Min Inter-Packet delta time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stream jitter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="mean_delta">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="max_delta">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="min_delta">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="jitter">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="min_max_rst">
        <property name="text">
         <string>Reset min/max jitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Debugger ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="self_id">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="rec_start">
          <property name="text">
           <string>Start Recording</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rec_stop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>End Recording</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
